CMakeLists.txt

```diff
@@ -28,27 +26,64 @@ if(HIGHFIVE_PARALLEL_HDF5)
 set(HDF5_PREFER_PARALLEL TRUE)
 endif()
 
-find_package(HDF5 REQUIRED)
+find_package(HDF5 QUIET REQUIRED)
 
 set(Boost_NO_BOOST_CMAKE TRUE)
 
 
 if(USE_BOOST)
-    find_package(Boost 1.41 REQUIRED COMPONENTS system unit_test_framework)
-    add_definitions(-DH5_USE_BOOST )
+  find_package(Boost 1.41 QUIET REQUIRED COMPONENTS system unit_test_framework)
+  add_definitions(-DH5_USE_BOOST )
 endif()
 
 if(HIGHFIVE_PARALLEL_HDF5)
-    find_package(MPI)
+  find_package(MPI)
 endif()
 
 install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/highfive
 		DESTINATION ${INCLUDE_INSTALL_DIR})
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include ${HDF5_INCLUDE_DIRS} ${Boost_INCLUDE_DIR} ${MPI_C_INCLUDE_PATH})
+# cmake target install
+if(CMAKE_MAJOR_VERSION GREATER 2) # No export(EXPORT) in CMake 2
```

Comment:
Hi Juan,

Sorry to bother you on that but two little points please :
- Can you make that disable by default and only enable with a -D option ?
- And if possible in a separated cmake file.

I know you guys love cmake targets for your git subprojects but I do not share this love a lot.

I would prefer the targets files to not invade every installation per default please